modules/network-observability-enriched-flows.adoc

```diff
@@ -6,10 +6,11 @@
 [id="network-observability-enriched-flows_{context}"]
 = Export enriched network flow data
 
-You can send network flows to Kafka, IPFIX, or both at the same time. Any processor or storage that supports Kafka or IPFIX input, such as Splunk, Elasticsearch, or Fluentd, can consume the enriched network flow data.
+You can send network flows to Kafka, IPFIX, the Red Hat build of OpenTelemetry, or all three at the same time. For Kafka or IPFIX, any processor or storage that supports those inputs, such as Splunk, Elasticsearch, or Fluentd, can consume the enriched network flow data.
```

Comment:
Actually, OpenTelemetry is a framework that can be present in any app such as Jaeger and Prometheus.
https://opentelemetry.io/docs/what-is-opentelemetry/

Netobserv allows to export either flows and / or metrics to a compatible OpenTelemetry endpoint; even if it's not a Red Hat product